src/page-modules/assistant/__tests__/assistant.test.tsx

```diff
@@ -11,7 +11,8 @@ import { GeocoderFeature } from '@atb/page-modules/departures';
 import { FeatureCategory } from '@atb/components/venue-icon';
 import { GeocoderApi } from '@atb/page-modules/departures/server/geocoder';
 import AssistantLayout from '../layout';
-import { NonTransitTripData } from '..';
+import { getQuayName, getStartModeAndPlace, NonTransitTripData } from '..';
```

Comment:
I think to keep APIs "clean"™ here, it is better to extract the helpers as its own module and test them instead of exporting them as part of external API.

So index.ts should be external APIs, but these are probably not something you want to export for all to see

Reply:
Or are these a part of the API of the module? 🤔 Might it then be a component?

Reply:
Can create a component of the header, and contain the functions within that component and have a separate test file for it. Is that what you meant?

Reply:
An open question more than anything, but does it make sense to instead of exporting the tools, you just export a component that handles it with an easier API. Low level building blocks vs Ready To Use limited component

Reply:
How about something like this, [dbebd0e](https://github.com/AtB-AS/planner-web/pull/49/commits/dbebd0e1d62e84b10194d4adb6ee5e90531f493f)?

I think a Ready To Use limited component is a good fit for this case.